packages/diagram_generator/linux/my_application.cc

```diff
@@ -1,7 +1,3 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
should we leave the license header in and add them to the other newly added files?

Reply:
Yes. Fixed.